extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-view.ts

```diff
@@ -48,8 +49,12 @@ export class DataExtensionsEditorView extends AbstractWebview<
   ToDataExtensionsEditorMessage,
   FromDataExtensionsEditorMessage
 > {
+  private static mostRecentlyActivePanel: WebviewPanel | undefined = undefined;
```

Comment:
I'm not sure whether a static variable is the best option here. We could also store this on the module as an instance variable, and I think that would behave more predictably. We would need some way to expose this information to the view, but I think we're doing something very similar for the variant analysis views.

Reply:
Ah, good idea. Yes, storing it on the module should be fine because there's only one module object. I'll move it to there, and then I guess we'll need to pass in a couple of method closures to allow the view to announce that it is now the active view, and check if it is still the most recently active.

Reply:
The `DataExtensionsEditorView` is gaining a number of new parameters, but it's for the best here I think. It's also something that should be easy to rewrite in the future if we change our minds.